code/game/objects/items/reagent_containers/hypospray.dm

```diff
@@ -70,6 +70,9 @@
 	if(!reagents.total_volume)
 		balloon_alert(user, "Hypospray is Empty.")
 		return
+	if(is_species(A, /datum/species/robot) || is_species(A, /datum/species/synthetic) || is_species(A, /datum/species/early_synthetic))
```

Comment:
Don't make snowflake like this.
These species have traits you can reference instead.

Reply:
Sure, I can use the IS_SYNTHETIC trait for checking synths, though I'm not sure what trait to use for combat robots.

Reply:
robots don't have synthetic afaik.
The more useful trait is the 'don't metabolise chems', whatever its actually called.

Reply:
Don't dead players recieve that trait though?

Reply:
no, they just stop processing

Reply:
Alright, it should now use flags to check instead of species datums.